src/e2e/java/teammates/e2e/pageobjects/StudentNotificationsPage.java

```diff
@@ -0,0 +1,120 @@
+package teammates.e2e.pageobjects;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import teammates.common.datatransfer.NotificationStyle;
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+
+/**
+ * Page Object Model for student notifications page.
+ */
+public class StudentNotificationsPage extends AppPage {
```

Comment:
As far as I can tell, this and `InstructorNotificationsPage` are duplicates of each other. There should be one base page object where both inherits from.

Reply:
Got it, will create another page object `UserNotificationsPage` and have `InstructorNotificationsPage` and `StudentNotificationsPage` inherit from it